### ISSUE TITLE
Beta issues

### ISSUE THREAD

**Issue by arghdos** [arghdos/spyJac#19]
Issues to be resolved before we can release the beta (copied from #18)

>- documentation (#16)
>- adding info to kernel objects (#14)
>- I'm still waiting on finalizing some of my additions to loo.py (this commit switches the required loo.py package to my own working branch)

>Things that would be nice to have, but aren't 100% critical:

>- Possibly adding a Cog-generated make-file based build system such that it is easier to build the interior kernels into a library separately from the wrapping code
>- Fix naming conventions / other flake errors
